doc/sphinx/using/libraries/funind.rst

```diff
@@ -242,39 +254,30 @@ Tactics
       This error may be raised when some inversion lemma failed to be generated by
       Function.
 
-
-   .. tacv:: functional inversion @natural
-
-      This does the same thing as :n:`intros until @natural` followed by
-      :n:`functional inversion @ident` where :token:`ident` is the
-      identifier for the last introduced hypothesis.
-
-   .. tacv:: functional inversion @ident @qualid
-             functional inversion @natural @qualid
-
-      If the hypothesis :token:`ident` (or :token:`natural`) has a type of the form
-      :n:`@qualid__1 {+ @term__i } = @qualid__2 {+ @term__j }` where
-      :n:`@qualid__1` and :n:`@qualid__2` are valid candidates to
-      functional inversion, this variant allows choosing which :token:`qualid`
-      is inverted.
-
 .. _functional-scheme:
 
 Generation of induction principles with ``Functional`` ``Scheme``
 -----------------------------------------------------------------
 
 
-.. cmd:: Functional Scheme @ident__0 := Induction for @ident' Sort @sort {* with @ident__i := Induction for @ident__i' Sort @sort}
+.. cmd:: Functional Scheme @fun_scheme_arg {* with @fun_scheme_arg }
+
+   .. old: Functional Scheme @ident__0 := Induction for @ident' Sort @sort {* with @ident__i := Induction for @ident__i' Sort @sort}
+
+   .. insertprodn fun_scheme_arg fun_scheme_arg
+
+   .. prodn::
+      fun_scheme_arg ::= @ident := Induction for @qualid Sort @sort_family
 
    This command is a high-level experimental tool for
-   generating automatically induction principles corresponding to
+   automatically generating induction principles corresponding to
    (possibly mutually recursive) functions. First, it must be made
-   available via ``Require Import FunInd``.
-   Each :n:`@ident__i` is a different mutually defined function
+   available via :cmd:`Require Import` ``FunInd``.
+   Each :n:`@ident` is a different mutually defined function
    name (the names must be in the same order as when they were defined). This
-   command generates the induction principle for each :n:`@ident__i`, following
+   command generates the induction principle for each :n:`@ident`, following
    the recursive structure and case analyses of the corresponding function
-   :n:`@ident__i'`.
+   :n:`@ident'`.
```

Comment:
There was a problem in the description before this PR. As you can see in https://coq.inria.fr/distrib/V8.7.2/refman/schemes.html#hevea_command240 (before the Sphinx migration), some `'` have been dropped. Everywhere the old version said `ident'`, the new version should say `qualid`.

Reply:
OK, only needed changes to `@qualid` in 2 places.

Reply:
In addition, the "for each" before `ident` is confusing and was introduced since this version I pointed (probably by someone who misinterpreted wha they read during the conversion). Maybe change to "This command generates the induction principles ident".

Reply:
I don't get you.  Each `with` clause has an `ident`.  I guess the `with` is to allow for mutual recursion.

Reply:
Don't forget this comment.

Reply:
Currently, the description contains "This command generates the induction principle for each ident". It's not super clear to me when I read this that "ident" is just the name given by the user to the generated induction principle.

Reply:
How's this?

```
   An experimental high-level tool that
   automatically generates induction principles corresponding to functions that
   may be mutually recursive.  The command generates an
   induction principle named :n:`@ident` for each given function named :n:`@qualid`.
   The :n:`@qualid`\s must be given in the same order as when they were defined.
   
   Note the command must be made available via :cmd:`Require Import` ``FunInd``.
```

Reply:
Looks good.